fcrepo-persistence-api/src/main/java/org/fcrepo/persistence/api/PersistentStorageFactory.java

```diff
@@ -18,9 +18,29 @@
 package org.fcrepo.persistence.api;
 
 /**
- * An interface that mediates CRUD operations to and from persistence storage.
+ * Factory class to create PersistentStorageSessions.
  *
- * @author dbernstein
+ * @author whikloj
+ * @since 2019-09-19
  */
-public interface PersistentStorage {
-}
\ No newline at end of file
+public abstract class PersistentStorageFactory {
```

Comment:
Shouldn't this be called PersistentStorageSessionFactory?   Also,  I think it might be easier from a testing perspective to have PersistentStorageSessionFactory  interface and a concrete implementation  (PersistentStorageSessionFactoryImpl)